specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -1351,6 +1433,28 @@
         }
       }
     },
+    "SapTablePartitionSettings": {
```

Comment:
For SapTableSource, could you please remove the two properties called rowSkips and batchSize? we would like to preserve them in the private swagger while do not want to expose them in the public swagger. Thanks.